static/bandh/js/main.js

```diff
@@ -1445,34 +1464,30 @@
         });
 
         if (isChannelPage) {
-            Filters.initialize();
+        	Filters.initialize();

```

Comment:
was the counter videos dynamic, but was removed because i found a way to get the total of videos of the channel